### PR TITLE
lxc/cluster_group: Added lxc cluster group add

### LINKED ISSUE #11726: Add a `lxc cluster group add` command
We currently have `lxc cluster group assign` which replaces the full set of groups for a member with a new set of groups.
We also have `lxc cluster group remove` which lets you remove a cluster member from a group.

But we don't have a `lxc cluster group add` to just add a cluster member into a specific group.

### PR DESCRIPTION
**Pull request by taltrums**
Fixes #11726
Hi, please review and let me know if any change is required.
Thanks

Screenshot-11

### CODE REVIEW

File: lxc/cluster_group.go [hunk 1]
```diff
@@ -595,3 +599,64 @@ func (c *cmdClusterGroupShow) Run(cmd *cobra.Command, args []string) error {
...
+		return err
+	}
+
+	resource := resources[0]
+
+	if resource.name == "" {
+		return fmt.Errorf(i18n.G("Missing cluster member name"))
+	}
+
+	// Add the cluster group
```
Comment (monstermunchkin) [inline lxc/cluster_group.go:640]:
```suggestion
	// Retrieve cluster member information.
```
Calling `GetClusterMember` doesn't add the cluster group.

**Review by monstermunchkin — CHANGES_REQUESTED** [review]
Just one small nit. Also, you need to add a commit with the updated translation files. You can update them using `make i18n`.

File: lxc/cluster_group.go (hunk 1 above)
Comment (taltrums) [reply lxc/cluster_group.go]: Hi @monstermunchkin, I have updated that comment and also updated the translation files as well. sorry for the hindsight.

**Comment by monstermunchkin** [comment]
Thanks. If you could just squash the first two commits into a single one, and change the title of the translation commit to something like `i18n: Update translations`. That'd be great.

Also, please sign off all your commits (you can do so with `git commit -s`), and rebase your changes as your branch apparently has conflicts.

**Comment by taltrums** [comment]
> Thanks. If you could just squash the first two commits into a single one, and change the title of the translation commit to something like `i18n: Update translations`. That'd be great.
> 
> Also, please sign off all your commits (you can do so with `git commit -s`), and rebase your changes as your branch apparently has conflicts.

@monstermunchkin, I  messed up my rebase, let me make a clean PR again. Thanks

**Pull request closed without merging**